src/convert/streams.ts

```diff
@@ -141,13 +141,14 @@ export class StandardWriter extends ComponentWriter {
     if (chunk.writeInfos.length !== 0) {
       try {
         const toResolve = new Set<string>();
+
         // it is a reasonable expectation that when a conversion call exits, the files of
         // every component has been written to the destination. This await ensures the microtask
         // queue is empty when that call exits and overall less memory is consumed.
         await Promise.all(
           chunk.writeInfos
             .map(makeWriteInfoAbsolute(this.rootDestination))
-            .filter(existsOrDoesntMatchIgnored(this.forceignore))
+            .filter(existsOrDoesntMatchIgnored(this.forceignore)) // Skip files matched by default ignore
```

Comment:
`existsOrDoesntMatchIgnored` should be getting the logger instance (`this.logger`) as its second argument here.